### PR TITLE
fix: Use correct AAB filename

### LINKED ISSUE #1783: Malformed aab file name
aab file name is wrong - eventyay-organizer-dev-aab

### PR DESCRIPTION
**Pull request by ShridharGoel**
Fixes #1783

### CODE REVIEW

File: scripts/update-apk.sh [hunk 1]
```diff
@@ -26,17 +26,21 @@ fi
...
-    done
-fi
+for file in app*; do
 
-if [ "$TRAVIS_BRANCH" == "$DEPLOY_BRANCH" ]; then
-    for file in app*; do
-          mv $file eventyay-organizer-dev-${file:4}
-    done
-fi
+    if [ ${file} =~ ".aab" ]; then
```
Comment (iamareebjamal) [inline scripts/update-apk.sh]: We need to add master/dev before it. aap == apk

Comment (ShridharGoel) [reply scripts/update-apk.sh]: Updated.

**Comment by iamareebjamal** [comment]
Change `/bin/rm -f eventyay-organizer-dev-*.apk` to `/bin/rm -f eventyay-organizer-dev-*`

**Comment by ShridharGoel** [comment]
@iamareebjamal Should the name be `eventyay-organizer-dev-app.aab` or `eventyay-organizer-dev.aab`

**Pull request merged**